ground/gcs/gcs.pri

```diff
@@ -67,6 +67,7 @@ macx {
     GCS_DATA_BASENAME = Resources
     GCS_DOC_PATH     = $$GCS_DATA_PATH/doc
     GCS_BIN_PATH     = $$GCS_APP_PATH/$${GCS_APP_TARGET}.app/Contents/MacOS
+    QMAKE_MACOSX_DEPLOYMENT_TARGET=10.9

```

Comment:
Yes.  This ends support for Mac OS X 10.7 and 10.8.  While I don't like dropping support for something that shipped 3 years ago (10.8 shipped in July 25, 2012), it's been unsupported since late 2013.   Basically all hardware that ran 10.7/10.8 can run 10.9 and 10.10.

There's a big installed base stuck on 10.6 because of hardware compatibility, but 10.7 and 10.8 are much more rare and falling much more quickly: https://stats.wikimedia.org/archive/squid_reports/2015-06/SquidReportOperatingSystems.htm
(down to .36+.36% from .39+.40% in the past month).